AutonomousDemo/Loop.cs

```diff
@@ -10,36 +10,26 @@ namespace AutonomousDemo
     public class Loop
     {
 
-        public void GameLoop(bool answer)
+        public void SimulationLoop(Vehicle vehicle)
```

Comment:
Probably should have specified you want a better name for the class itself. ;)

Maybe call the class "SimulationLoop" and this method "Run"?